gradle/checkstyle.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'checkstyle'
+
+task checkstyle(type: Checkstyle) {
+    description 'Analyzing project and creating report with bad things in the code'
+
+    ignoreFailures = false
+    configFile rootProject.file('checkstyle/checkstyle.xml')
+
+    source 'src'
+    include '**/*.java'
+
+    classpath = files()
+}
+
+checkstyle {
+    toolVersion = '7.7'
```

Comment:
Had to update Checkstyle and remove one rule https://discuss.gradle.org/t/checkstyle-checker-cannot-be-created-with-gradle-3-5/22474/2